src/Microsoft.ML.Data/Transforms/KeyToValue.cs

```diff
@@ -109,9 +108,7 @@ public static IDataTransform Create(IHostEnvironment env, Arguments args, IDataV
             return transformer.MakeDataTransform(input);
         }
 
-        /// <summary>
-        /// Factory method for SignatureLoadModel.
-        /// </summary>
+        // Factory method for SignatureLoadModel.
```

Comment:
>// Factory method for SignatureLoadModel. [](start = 8, length = 41)

I see you doing this multiple places, is there any reason for it?

If we instead *kept* it an XML comment, we could improve this by having a `<see` tag on it, which overall seems like a good idea. We're not doing this now, but it seems like an unforced regression into a worse, not a better, place.

Reply:
only reason why we had <summary> tag because it was requirement for public method, and since it's no longer public I just prefer mark this methods as comments since 
a) This is most common pattern in our codebase.
b) I think about this as more as internal message for developer: "Hey, don't delete this one, even if no one uses it, we actually call it via our DI system." Summary is more like a description of what is going on in this method, and this is more like a mark/internal comment.

---
In reply to: [245456969](https://github.com/dotnet/machinelearning/pull/2031#discussion_r245456969) [](ancestors = 245456969)

Reply:
<summary> tags are a requirement for public methods. That ***does not make them a bad thing for private or internal methods***, especially if those comments explicitly reference other structures in the codebase, as this does.

Why would a *developer* prefer this comment, to the old comment? If you had instead had a `<see` tag, as suggested above, that developer could even F12 on it to see what the heck we're talking about. Right?

---
In reply to: [245457620](https://github.com/dotnet/machinelearning/pull/2031#discussion_r245457620) [](ancestors = 245457620,245456969)

Reply:
This should probably just be a simple search-replace, I think, to put it into proper form. At least, so I'd think.

---
In reply to: [245458497](https://github.com/dotnet/machinelearning/pull/2031#discussion_r245458497) [](ancestors = 245458497,245457620,245456969)

Reply:
I know I can't win argument with you so I reverted my changes to summary comments. Although I'm still adding comments for other places since this is common pattern in our code.
I hope this is not a blocker, since main intent of this PR is to hide IDataTransforms methods.

---
In reply to: [245459044](https://github.com/dotnet/machinelearning/pull/2031#discussion_r245459044) [](ancestors = 245459044,245458497,245457620,245456969)